config/routes.rb

```diff
@@ -1,8 +1,7 @@
 Rails.application.routes.draw do
   devise_for :users
 
-  root to: 'pages#home'
```

Comment:
If we don't need  pages#home anymore we should drop the [pages controller](https://github.com/WSTI-students/waterme/blob/master/app/controllers/pages_controller.rb#L1) and it's [view template](https://github.com/WSTI-students/waterme/blob/master/app/views/pages/home.html.erb)